.classpath

```diff
@@ -23,10 +23,6 @@
 			<attribute name="org.eclipse.jst.component.dependency" value="/WEB-INF/lib"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="lib" path="C:/Users/Cihan/Downloads/mysql-connector-java-5.1.19-bin.jar">
```

Comment:
This path will be different for each of us, but ill introduce a change that will be universal with my merge